internal/network/machine_network_cidr.go

```diff
@@ -255,6 +255,25 @@ func GetMachineCidrForUserManagedNetwork(cluster *common.Cluster, log logrus.Fie
 	return ""
 }
 
+func GetIpForSingleNodeInstallation(cluster *common.Cluster, log logrus.FieldLogger) (string, error) {
+	bootstrap := common.GetBootstrapHost(cluster)
+	if bootstrap == nil {
+		return "", errors.Errorf("no bootstrap host were found in cluster")
+	}
+	cidr := GetMachineCidrForUserManagedNetwork(cluster, log)
+	cluster.MachineNetworkCidr = cidr
```

Comment:
Why is this here?
Seems irrelevant to what this function says it does

Reply:
this is the way to provide machineCidr to the next one